apps/web/src/services/asset-inventory/collector/shared/collector-forms/PluginMetadataForm.vue

```diff
@@ -44,13 +48,12 @@ import {
 const collectorFormStore = useCollectorFormStore();
 const collectorFormState = collectorFormStore.$state;
 
-const emit = defineEmits<{(e: 'change:isSchemaFormValid', isValid: boolean): void;}>();
+const emit = defineEmits<{(e: 'change:isValid', isValid: boolean): void;}>();
```

Comment:
suggestion: It is advantageous to use 'update:' prefix for reasons below:
1. Developers can use 'sync' of Vue2 or multi 'v-model' of Vue3.
2. Developers can guess very easily that this event parameter is only the value for the specific prop. (in this case, 'isValid')
3. Less naming convention help reducing human errors.